<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Lukáš Tinkl &lt;ltinkl@redhat.com&gt;</author>
 <class>BondWidget</class>
 <widget class="QWidget" name="BondWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>417</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Interface name:</string>
     </property>
     <property name="buddy">
      <cstring>ifaceName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KLineEdit" name="ifaceName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Bonded connections:</string>
     </property>
     <property name="buddy">
      <cstring>bonds</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="4">
      <widget class="QListWidget" name="bonds">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Add…</string>
       </property>
       <property name="icon">
        <iconset theme="list-add"/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit…</string>
       </property>
       <property name="icon">
        <iconset theme="configure"/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnDelete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset theme="edit-delete"/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Mode:</string>
     </property>
     <property name="buddy">
      <cstring>mode</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KComboBox" name="mode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Link monitoring:</string>
     </property>
     <property name="buddy">
      <cstring>linkMonitoring</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="KComboBox" name="linkMonitoring">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Monitoring frequency:</string>
     </property>
     <property name="buddy">
      <cstring>monitorFreq</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="monitorFreq">
     <property name="suffix">
      <string> ms</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QStackedWidget" name="monitorStack">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="miiPage">
      <layout class="QFormLayout" name="formLayout">
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Link up delay:</string>
         </property>
         <property name="buddy">
          <cstring>upDelay</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="upDelay">
         <property name="suffix">
          <string> ms</string>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Link down delay:</string>
         </property>
         <property name="buddy">
          <cstring>downDelay</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="downDelay">
         <property name="suffix">
          <string> ms</string>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="arpPage">
      <layout class="QFormLayout" name="formLayout_3">
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>ARP targets:</string>
         </property>
         <property name="buddy">
          <cstring>arpTargets</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KLineEdit" name="arpTargets">
         <property name="toolTip">
          <string>An IP address or comma-separated list of addresses.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ifaceName</tabstop>
  <tabstop>bonds</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>mode</tabstop>
  <tabstop>linkMonitoring</tabstop>
  <tabstop>monitorFreq</tabstop>
  <tabstop>upDelay</tabstop>
  <tabstop>downDelay</tabstop>
  <tabstop>arpTargets</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>linkMonitoring</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>monitorStack</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
